scss/core/base.scss

```diff
@@ -397,42 +397,6 @@ html.no-touchevents.js-focus-visible {
     &:focus-visible {
       box-shadow: 0px 0px 0px $focusOutlineWidth $focusOutlineColor;
     }
-
-    @include above($tabletBreakpoint) {
-      &:focus {
-        box-shadow: 0px 0px 0px $focusOutlineWidthTablet $focusOutlineColorTablet !important;
```

Comment:
@YaroslavOvdii If these are removed, where would the tablet and desktop settings be used? Do they never get used?

If so, this creates a bug where users would configure a setting in Studio for tablet/desktop and the configurations won't work in the app.

Reply:
@tonytlwu Yes, you are right. But I've made these changes because we would have this [issue](https://share.getcloudapp.com/KoudjGqE). It appeared because in the core CSS file we have added styles only for mobile devices (without breakpoints).
Could we add breakpoints to the core CSS file?

Reply:
@YaroslavOvdii Yes, I can add breakpoints to the core CSS file. Though, note that the CSS declarations will be loaded in quite a weird order.

```css
/* core.css */

.selector {
  box-shadow: none;
}

@media screen and (min-width: x) {
  .selector {
    box-shadow: none;
  }
}

@media screen and (min-width: y) {
  .selector {
    box-shadow: none;
  }
}

/* theme.css */

.selector {
  box-shadow: 0px 0px 0px 1px red;
}

@media screen and (min-width: x) {
  .selector {
    box-shadow: 0px 0px 0px 1px red;
  }
}

@media screen and (min-width: y) {
  .selector {
    box-shadow: 0px 0px 0px 1px red;
  }
}
```

Will this produce the result that you want? You might want to test it before I make the changes, and then tell me specifically where you'd like me to make the changes.

Remember, the priorities are:

1. It must work -- otherwise it's a bug
2. Avoid any variables to be unused -- otherwise it's a bug
3. Avoid CSS declarations in .scss files that don't use any variables -- to minimise theme.css file size

Reply:
@tonytlwu Yes, this should do the desired result for this task. Could you please add this to development first so I could test it and confirm that it works in the way we want?

Reply:
@YaroslavOvdii I've deployed this to development.

![image](https://user-images.githubusercontent.com/290733/117347124-943da380-aea0-11eb-9644-784c30619a04.png)

Let me know if this works.

Reply:
@tonytlwu Yes, it's working. And as I can see everything works as it should be.
[Screencast](https://share.getcloudapp.com/9ZuxpwoO)
Also, should I create a new PR to remove scss code without variables or do it in this one?

Reply:
@tonytlwu Removed code that doesn't use variables.
Also a new [screencast ](https://share.getcloudapp.com/2NuwZYwn) with different breakpoints.